README.md

```diff
@@ -187,17 +188,57 @@ logger:
 ```
 and after the problem occurs please create an issue with the log (`/config/home-assistant.log`).
 
+## Lights are not responding or turning on by themselves
+
+This addon sends many more commands to lights compared to what humans would typically send. If the network used to send light commands is not healthy:
+
+- Manual commands like turning lights on or off may feel laggy.
+- Lights may not respond to commands at all.
+- Home Assistant may think a light is on, when it's actually off. Adaptive Lights will send it's regular adjustments causing the light to turn on after it's turned off.
+
+What's important is that many bugs that seem to be caused by this integration are really due to other unrelated issues. Fixing those will make your Home Assistant experience much better. Consider this integration a great stress test of your Home Assistant setup!
+
+### Wifi networks
+
+Make sure bulbs have a solid connection to your Wifi network. In general, if the signal is less than -70dBm, the connection is weak and may drop messages.
+
+### Zigbee, Z-Wave, and other mesh networks
+
+These types of mesh networks usually need powered devices that act as routers (that repeat messages) back to the central coordinator (the radio connected to Home Assistant). Most Philips lights are routers, but Ikea, Sengled, and generic Tuya bulbs often are not. If devices become unavailable or miss responding to commands, Adaptive Lighting will only make things worse. Use reporting tools such as network maps (available in ZHA, zigbee2mqtt, deCONZ, and ZWaveJS UI) to check your network. Smart plugs are often a cost-effective way to add additional routers to your network.
+
+For Zigbee networks, groups are **absolutely required for good performance**. For example, imagine you want to use Adaptive Lighting in a hallway with 6 bulbs. If you add each individual bulb in the Adaptive Lighting configuration, then six individual commands will be sent to adjust them, which can eventually overwhelm a network. Instead, create a group in your Zigbee software (but _not_ a regular Home Assistant group), and add the one group to the Adaptive Lighting configuration. This will send only a single broadcast command to adjust the bulbs, giving much better response times and keeping the bulbs adjusting in sync with each other.
```

Comment:
I actually believe that one should **not** use Zigbee groups but Home Assistant groups instead, not to reduce congestion, but because it leads to fewer sudden turn ons.

Reply:
Interesting! That doesn't match my own experience. Even with two bulbs in a fixture, there would be obvious lag in sending a "turn on" event to bulbs individually versus in a Zigbee group.

A few external references:

* Phoscon won't let you control lights at all unless they are in at least one group, to encourage use of groups over individual lights: https://phoscon.de/en/app/doc#group-edit-lights-en
* Hubitat specifically calls out using groups for better performance: https://docs2.hubitat.com/apps/groups-and-scenes: "Zigbee Group Messaging option allows you to group supported Zigbee lights together to eliminate the "popcorn effect" when you turn them on or adjust levels."
* zigbee2mqtt (which I use) has some handy options to control when groups show up as on or off when members have different states. I haven't had issues with lights turning on (other than the transition bug I wrote about) because I nearly always expose groups only to my HA dashboard and HomeKit: https://www.zigbee2mqtt.io/guide/usage/groups.html

So perhaps this could be written as something like:

```suggestion
For most Zigbee networks, groups are **absolutely required for good performance**. For example, imagine you want to use Adaptive Lighting in a hallway with 6 bulbs. If you add each individual bulb in the Adaptive Lighting configuration, then six individual commands will be sent to adjust them, which can eventually overwhelm a network. Instead, create a group in your Zigbee software (but _not_ a regular Home Assistant group), and add the one group to the Adaptive Lighting configuration. This will send only a single broadcast command to adjust the bulbs, giving much better response times and keeping the bulbs adjusting in sync with each other.

A good rule to follow is that if you always control lights together (like bulbs in a ceiling fixture), then they should be in a Zigbee group. Then, only expose the group (and not individual bulbs) in Home Assistant Dashboards and external systems like Google Home or Apple HomeKit.
```